cypress.json

```diff
@@ -2,8 +2,9 @@
   "projectId": "Testing Automation",
```

Comment:
we should need create other one to the new version of ghost and test it

Reply:
I'm not sure if we could use different projectId with the same source code.